src/components/TextField/TextField.tsx

```diff
@@ -405,6 +418,7 @@ export function TextField({
     maxLength,
     spellCheck,
     pattern,
+    inputMode,
```

Comment:
I’d argue this should default to `decimal` on `number` inputs. I think it’s only iOS Safari that doesn’t show the right keyboard by default on `number` inputs, though.

Reply:
I don't think there is a "safe" default for input type `number` as even the broader `decimal` compared to `numeric`  mentions "Devices may or may not show a minus key (-)." 😬   [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode).

Reply:
Oh damn.